src/referentielijsten/api/tests/test_tabellen.py

```diff
@@ -71,3 +75,16 @@ def test_tabellen_is_not_geldig(self):
         self.assertEqual(response_data["count"], 1)
         self.assertEqual(response_data["results"][0]["code"], "2")
         self.assertEqual(response_data["results"][0]["naam"], "ongeldig")
+
+    def test_items_link_returns_correct_admin_url(self):
```

Comment:
This test file is specifically for API tests for tabellen, so I think it'd be be better to create a separate file `test_tabellen_admin.py` with a testcase that inherits from `WebTest`. With webtest you can do something like `self.app.get(reverse(expected_url), user=user)` (see `src/referentielijsten/accounts/tests/test_permission_limit.py` for example)